### DISCUSSION TITLE
Image v3 Comp - srcset attribute and other new features

### DISCUSSION THREAD

**Discussion by Tylopilus** [adobe/aem-core-wcm-components#1708] [Q&A]
Hi guys, just wondering what does an editor actually see when he uses the image v3 comp vs image v2 comp?

I have added this to my local project and when I am switching to the v3 comp I cannot see any difference (even in the html-output).

What I mean is:
* I can only define 1 image, not a srcset (or is it dynamic media only?)
* I don't see the width and height attributes in the final html (js enabled)
* lazy loading is not done natively with loading=lazy but still using JS

Arent these features built in with the release of v3?

I am especially interested in defining a srcset-attribute to display different images based on the viewport-width.

**Comment by DigiLiviu** [comment]
Related to the srcset, since we upgraded to image v3 and the latest AEMaaCS SDK, we've been getting the following errors in our Custom Functional Tests :

```
Caused by: java.net.URISyntaxException: Illegal character in query at index 113: /adobe/dynamicmedia/deliver/dm-aid--2dc846e3-59e9-48d9-92c0-397d20c735e3/hp-hero-video.png?preferwebp=true&width={width}&quality=85
```

That value is coming from the `data-cmp-src` attribute which seems to act as a sort of template for the src set values but the `{width}` placeholder is triggering an error as our Custom Functional Tests are validating the page and connecting with an attribute value that has a `{` character, which its now deeming illegal...
